mautic/mautic
# fix test payloads for lead webhooks

| Q  | A
| --- | ---
| Bug fix? | Yes |
| New feature? | No |
| Automated tests included? | No |
| Related user documentation PR URL | No |
| Related developer documentation PR URL | No |
| Issues addressed (#s or URLs) | #5119 |
| BC breaks? | No |
| Deprecations? | No |

#### Description:
Like explained in #5119 event name is present twice in webhook **test payload**. This doesn't reflect the real payload sent on this events.

#### Steps to reproduce the bug:
1. Create a new webhook for leads events
2. Click on "Send test payload"
3. Each event name will appear twice in the payload 

#### Steps to test this PR:
1. Create a new webhook for leads events
2. Click on "Send test payload"
3. Each event name will appear only once
